javascript/javascript1/week3/javascript.js

```diff
@@ -0,0 +1,130 @@
+// Remove name from array list
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+];
+const nameToRemove = "Ahmad";
+for (var i = names.length - 1; i >= 0; i--) {
+    if (names[i] === nameToRemove) {
+        names.splice(i, 1);
+    }
+}
+console.log(names);
+
+// Travel time Information function
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+}
+function notThisFunctionName(travelInformation) {
+    const travelTime = travelInformation.destinationDistance / travelInformation.speed;
+    return travelTime;
+}
+const travelTime = notThisFunctionName(travelInformation);
+console.log(travelTime)
+
+const hour = Math.floor(travelTime);
+let decpart = travelTime - hour;
+const min = 1 / 60;
+decpart = min * Math.round(decpart / min);
+const minute = Math.floor(decpart * 60) + '';
+console.log(`${hour} hours and ${minute} minutes`)
+
+// Time Calculation of  series Duration in Life. 
+const seriesDurations = [
+    {
+        title: "Game of thrones",
+        days: 3,
+        hours: 1,
+        minutes: 0,
+    },
+    {
+        title: "Sopranos",
+        days: 6,
+        hours: 15,
+        minutes: 0,
+    },
+    {
+        title: "The Wire",
+        days: 9,
+        hours: 3,
+        minutes: 0,
+    },
+];
+
+function logOutSeriesText() {
+    // write code here
+    let totalPerLifeSpan = 0;
+    for (let i = 0; i < seriesDurations.length; i++) {
+        const age = 80;
+        const serialTime = seriesDurations[i].days + (seriesDurations[i].hours / 24);
+        const lifeLongSpan = age * 365;
+        const perLifeSpan = serialTime * 100 / lifeLongSpan;
+        console.log(`${seriesDurations[i].title} took ${perLifeSpan.toFixed(3)}% of my life`)
+        totalPerLifeSpan += +perLifeSpan.toFixed(3)
+    }
+    // Return total percentage of life Span 
+    return console.log(`In total that is ${totalPerLifeSpan} of my life`)
+}
+const perLifeSpan = logOutSeriesText(seriesDurations)
+
```

Comment:
//would i like to know why did you pass the parameter to function, and what is the use of 'perlifeSpan' variable here. 
//you can simply call function like  'logOutSeriesText()';
I checked the perlifeSpan in console but it shows undefined , if  i am wrong let me know i will also look at it  again how it works . 
thank you prakash .

Reply:
oky... I have used parameter to get value from arrays.   could you please check now let me know I have updated code.  
you are right.  I can get value without passing parameter as well.